packages/valory/skills/hello_world_abci/rounds.py

```diff
@@ -132,14 +131,15 @@ class PrintMessageRound(CollectDifferentUntilAllRound, HelloWorldABCIAbstractRou
     allowed_tx_type = PrintMessagePayload.transaction_type
     payload_attribute = get_name(PrintMessagePayload.message)
 
-    # TODO: what's the point of this? It might be better
-    # to store the messages on the synchronized state so
-    # the dev actually learns something (see todo above)
     def end_block(self) -> Optional[Tuple[BaseSynchronizedData, Event]]:
         """Process the end of the block."""
         if self.collection_threshold_reached:
             synchronized_data = self.synchronized_data.update(
                 participants=self.collection,
+                printed_messages=[
+                    cast(PrintMessagePayload, payload).message
+                    for payload in self.collection.values()
+                ],
```

Comment:
So we now store the message in the db, and we can retrieve it as well, but this is not used in the behaviours right? Is this just for educational purposes then?